tests/mocks/addons/new-payment-method/.gitignore

```diff
@@ -1,5 +1,7 @@
 # auto-generated/build-artifact files that should not be committed
 core/third_party_libs/dompdf/lib/fonts/log.htm
+core/third_party_libs/dompdf/lib/fonts/*.afm.php
+core/third_party_libs/dompdf/lib/fonts/*.ufm.php
```

Comment:
these lines in `.gitignore` were actually in there by accident- the `new-payment-method` add-on doesn't have a `core/third_party_libs/dompdf/lib/fonts` directory anyway. So these lines may as well be removed from all the skeleton add-ons' `.gitignore` files.